src/components/bookItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useDispatch } from "react-redux";
+import { addToCart } from "../../routes/cart/cart.reducer";
+import type { AppDispatch } from "../../store/store";
+import { Book } from "../../types";
+import "./bookItem.styles.css";
+
+export const BookItem = (book: Book): JSX.Element => {
+
+  const dispatch: AppDispatch = useDispatch();
+  const { bookTitle, bookAuthor, bookDescription, bookPrice, bookImgUrl } = book;
```

Comment:
Use prop destructuring in the fn definition itself.

Reply:
Sure Sir adding these changes